incubator/couchdb/templates/statefulset.yaml

```diff
@@ -133,7 +133,7 @@ spec:
         name: database-storage
         labels:
           app: {{ template "couchdb.name" . }}
-          chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+          chart: {{ .Chart.Name }}
```

Comment:
`chart` and `heritage` labels should be removed here.

Reply:
Done, completely removed chart label and heritage one as well.